backend/src/index.ts

```diff
@@ -110,7 +106,7 @@ app.post(
         email,
         password: hashedPassword,
         salt,
-        userType,
+        userType:"ATTENDEE",

```

Comment:
note to self: perhaps this can be removed from the database
(dw abt this ricky)